jaggr-core/src/main/java/com/ibm/jaggr/core/impl/resource/JsxResourceConverter.java

```diff
@@ -527,7 +532,7 @@ public Scriptable call() throws Exception {
 							threadScope.setParentScope(null);
 							// require in the transformer and extract the transform function
 							Require require = builder.createRequire(ctx, threadScope);
-							Scriptable jsxTransformInstance = require.requireMain(ctx, JSXTRANSFORMER_NAME);
+							Scriptable jsxTransformInstance = require.requireMain(ctx, xformerModuleName);

```

Comment:
It is also used as the name of the init-param that specifies the transformer URI